integration-tests/maven/src/test/java/io/quarkus/maven/it/AddExtensionIT.java

```diff
@@ -141,6 +141,7 @@ private void addExtension(boolean plural, String ext)
         }
         enableDevToolsTestConfig(properties);
         request.setProperties(properties);
+        passUserSettings(request);
```

Comment:
Instead of having to remember to call this method in each test, what about wrapping the `DefaultInvoker` in `MojoTestBase.initInvoker(File)` so that the user settings are passed automatically in `Invoker.execute(InvocationRequest)`?